cpp-client/deephaven/dhclient/include/private/deephaven/client/arrowutil/arrow_column_source.h

```diff
@@ -57,11 +57,11 @@ class GenericBackingStore {
       auto element = (*array_)[i];
       bool is_null;
       if (element.has_value()) {
-        ArrowValueConverter::Convert(*element, dest);
+        ArrowValueConverter::Convert(*element, dest++);
         is_null = false;
       } else {
         // placeholder
-        *dest = ElementType();
+        *dest++ = ElementType();
         is_null = true;
       }
```

Comment:
Maybe it would be cleaner to just to `++dest` right after the `if {} else {}` and remove the postfix ++ on dest inside of it?

Reply:
Done